src/pages/Home.js

```diff
@@ -18,6 +18,16 @@ function Home() {
       image: `${process.env.PUBLIC_URL}/assets/525661.png`,
       url: "/series",
     },
+    {
+      type: "Hollywood Mix",
+      image: `${process.env.PUBLIC_URL}/assets/525661.png`,
+      url: "/hollywood",
+    },
+    {
+      type: "Bollywood Mix",
+      image: `${process.env.PUBLIC_URL}/assets/525661.png`,
+      url: "/bollywood",
```

Comment:
Have you made these routes in App.js?

Reply:
Updated it in my next commit but not updated the component part.

Reply:
As those pages are not made yet.